resolwe/flow/tests/test_models.py

```diff
@@ -521,6 +524,197 @@ def test_raw_process(self):
         self.assertNotEqual(response.data['id'], self.data.pk)
 
 
+class DuplicateTestCase(TestCase):
+
+    def test_data_duplicate(self):
+        process1 = Process.objects.create(
+            contributor=self.user,
+            type='data:test:first:',
+        )
+        process2 = Process.objects.create(
+            contributor=self.user,
+            input_schema=[
+                {'name': 'data_field', 'type': 'data:test:first:'},
+            ],
+            output_schema=[
+                {'name': 'json_field', 'type': 'basic:json:'},
+            ],
+        )
+
+        data = Data.objects.create(
```

Comment:
Name this one `child_data` or `input_data` and the other one just `data` to make it cleaner.